### PR TITLE
Styling options for pie labels

### PR DESCRIPTION
**Pull request by rmenegaux**
Added the following options:
- `display_labels`: whether to make the labels visible or not
- `label_color`
- `font_size`
- `font_weight`

Fixes issue #99, helps with #130

### CODE REVIEW

**Comment by dmadeka** [comment]
As per conversation, @rmenegaux, the notebook does not work and needs to be improved

File: examples/Pie.ipynb [hunk 1]
```diff
@@ -28,20 +28,21 @@
    "outputs": [],
```
Comment (dmadeka) [inline examples/Pie.ipynb]: Lets add a markdown cell above that says "Here we generate the simplest possible pie chart by passing the data as the sizes we want each slice to be"?

File: examples/Pie.ipynb [hunk 2]
```diff
@@ -28,20 +28,21 @@
...
    "execution_count": null,
    "metadata": {
-    "collapsed": true
+    "collapsed": false
    },
    "outputs": [],
    "source": [
-    "pie.selected_style = {\"opacity\": \"1\", \"stroke\": \"white\", \"stroke-width\": \"2\"}\n",
-    "pie.unselected_style = {\"opacity\": \"0.2\"}"
+    "# Modify pie data dynamically\n",
```
Comment (dmadeka) [inline examples/Pie.ipynb]: Lets make these `####` Markdown cells?

File: examples/Pie.ipynb [hunk 3]
```diff
@@ -165,10 +204,22 @@
    },
    "outputs": [],
    "source": [
+    "from bqplot import ColorScale, ColorAxis\n",
```
Comment (dmadeka) [inline examples/Pie.ipynb]: This needs more documentation I guess, and separate cells

File: examples/Pie.ipynb [hunk 4]
```diff
@@ -179,38 +230,30 @@
...
-   "outputs": [],
-   "source": [
-    "Figure(marks=[line] + pies, axes=[ax_x, ax_y])"
+    "pies = [Pie(sizes=precipit, x=date, y=temp, \n",
+    "            scales={\"x\": sc_x, \"y\": sc_y}, radius=30., stroke='navy',\n",
+    "            colors=['navy', 'navy'], opacities=[1, .1]) \n",
+    "        for precipit, date, temp in zip(avg_precipitation_days, dates, temperatures)]\n",
+    "\n",
+    "Figure(title='Kathmandu precipitation', marks=pies, axes=[ax_x, ax_y],\n",
+    "       padding_x=.05, padding_y=.1)"
```
Comment (dmadeka) [inline examples/Pie.ipynb]: very cool graph

**Comment by dmadeka** [comment]
@rmenegaux The pie labels color example doesnt work for me. And if I run the last cell, and then re run the previous ones, it throws an error. Also do we need to import the entire numpy? I think you only use 2 or 3 functions.

**Comment by rmenegaux** [comment]
@dmadeka turns out this is a more general bug that will have to be dealt with in another PR. 
Otherwise this is rebased and ready to go!

**Comment by dmadeka** [comment]
@rmenegaux There's an empty cell in the bottom, delete that and we're all set

**Comment by rmenegaux** [comment]
@dmadeka done!

**Pull request merged**